index.js

```diff
@@ -17,7 +18,7 @@ const { helpers: androidHelpers, SETTINGS_HELPER_PKG_ID } = androidHelperIndex;
 
 export default AndroidDriver;
 export {
-  androidHelpers, androidCommands, AndroidDriver,
+  androidHelpers, androidCommands, AndroidDriver, newMethodMap,
```

Comment:
do we need to export it here? I assume it should be enough to have the map being statically exposed by the driver class

Reply:
yeah, good point, i didn't know if it would work given how things are transpiled. but it seems to work.